src/components/App.vue

```diff
@@ -2,6 +2,7 @@
   <div id="root">
     <side-board id="side-board" />
     <card-board id="card-board" />
+    <admin-board id="admin-board" />
```

Comment:
I haven't had a chance to look at the whole PR yet, but I would suggest putting a v-if on this with a feature-flag. That way you can merge this into develop without it showing up yet. Just for clarity: when I say "feature-flag" I just mean some boolean somewhere that is hardcoded to `false`